Gems/Camera/Code/Source/CameraComponentController.cpp

```diff
@@ -280,6 +299,28 @@ namespace Camera
         DeactivateAtomView();
     }
 
+    void CameraComponentController::CreateRenderPipelineForTexture()
+    {
+        auto scene = AZ::RPI::RPISystemInterface::Get()->GetSceneByName(AZ::Name("Main"));
+
+        const AZStd::string pipelineName = "Camera_" + m_entityId.ToString() + "_Pipeline";
+
+        auto defaultPipelineDesc = scene->GetDefaultRenderPipeline()->GetDescriptor();
+
+        // Use default render pipeline for the camera render pipeline
+        // We can add function to use customized render pipeline for camera in the future
```

Comment:
That would be cool

Reply:
It's actually using a root pass template which can be specified in `m_config.m_pipelineTemplate`. So user can specify other template other than the one in Atom common feature gem.

Reply:
I removed the comment.